android/app/src/main/java/org/fossasia/openevent/activities/auth/SignUpActivity.java

```diff
@@ -61,6 +64,8 @@ protected void onCreate(Bundle savedInstanceState) {
         createPasswordInput = (TextInputEditText) createPasswordWrapper.getEditText();
         confirmPasswordInput = (TextInputEditText) confirmPasswordWrapper.getEditText();
 
+        signUpActivityViewModel = ViewModelProviders.of(this).get(SignUpActivityViewModel.class);
+
```

Comment:
Please remove this space

Reply:
@simarsingh24 The aim is to make things clearer to the reader. If you observe the whole method closely(the parts that are left untouched in this PR), you will realise that spaces have been provided between unrelated things. For eg:

```Java

setSupportActionBar(toolbar);
getSupportActionBar().setDisplayHomeAsUpEnabled(true);

switchToLogin.setOnClickListener(v -> startActivity(new Intent(SignUpActivity.this, LoginActivity.class)));
```

So to remain consistent with what has been done, i think it's better to add a space as that line is unrelated to anything above/below it.

Reply:
if that's the idea then its fine :)